### ISSUE TITLE
Building from source fails

### ISSUE THREAD

**Issue by 7sharp9** [avh4/elm-format#206]
```
cabal: Could not resolve dependencies:
trying: elm-format-0.3.2 (user goal)
trying: transformers-0.5.2.0/installed-0.5... (dependency of
regex-applicative-0.3.3)
next goal: optparse-applicative (dependency of elm-format-0.3.2)
rejecting: optparse-applicative-0.12.1.0, optparse-applicative-0.12.0.0
(conflict: elm-format => optparse-applicative>=0.11 && <0.12)
rejecting: optparse-applicative-0.11.0.2, optparse-applicative-0.11.0.1,
optparse-applicative-0.11.0 (conflict: transformers==0.5.2.0/installed-0.5...,
optparse-applicative => transformers>=0.2 && <0.5)
rejecting: optparse-applicative-0.10.0, optparse-applicative-0.9.1.1,
optparse-applicative-0.9.1, optparse-applicative-0.9.0,
optparse-applicative-0.8.1, optparse-applicative-0.8.0.1,
optparse-applicative-0.8.0, optparse-applicative-0.7.0.2,
optparse-applicative-0.7.0.1, optparse-applicative-0.7.0,
optparse-applicative-0.6.0, optparse-applicative-0.5.2.1,
optparse-applicative-0.5.2, optparse-applicative-0.5.1,
optparse-applicative-0.5.0, optparse-applicative-0.4.3,
optparse-applicative-0.4.2, optparse-applicative-0.4.1,
optparse-applicative-0.4.0, optparse-applicative-0.3.2,
optparse-applicative-0.3.1, optparse-applicative-0.3.0,
optparse-applicative-0.2.0, optparse-applicative-0.1.1,
optparse-applicative-0.1.0, optparse-applicative-0.0.1 (conflict: elm-format
=> optparse-applicative>=0.11 && <0.12)
Dependency tree exhaustively searched.
```

**Comment by avh4** [comment]
Are you using ghc 7.10.3?

**Comment by 7sharp9** [comment]
Yes

**Comment by avh4** [comment]
Hmm, looks like maybe we need to upgrade to optparse-applicative-0.12 ?

**Comment by avh4** [comment]
Hmm... are you doing these steps from the README?

```bash
cabal update

git clone https://github.com/avh4/elm-format.git
cd elm-format
cabal sandbox init
cabal install --only-dependencies --enable-tests
cabal build
./dist/build/elm-format-0.17/elm-format-0.17 --help
```

I just tried that in a clean folder and it worked for me, and used optparse-applicative-0.11.0.2.  I'm not sure what the problem here is.

**Comment by paparga** [comment]
I've never than anything in Haskell before, I just saw help wanted. 

I successfully build it following the steps from README, but only after running `cabal update` first, that it is not included.

OS:  Ubuntu 16.04
GHC: 7.10.3

**Comment by avh4** [comment]
I've just update to use `stack` instead of `cabal`.  Please update to the lastest version of `master`, and try the new instructions at https://github.com/avh4/elm-format#development-info

**Comment by avh4** [comment]
Building from source is now down with `stack`.  Open a new issue if there are still problems.
